src/instance.ts

```diff
@@ -41,7 +129,13 @@ import {Database} from './database';
  * const instance = spanner.instance('my-instance');
  */
 class Instance extends common.ServiceObject {
-  constructor(spanner, name) {
+  formattedName_: string;
+  // tslint:disable-next-line: no-any
+  request: (config: any, callback: Function) => void;
+  requestStream: (config?: {}) => Stream;
```

Comment:
`requestStream` returns a `Duplex` stream, is the `Stream` type compatible?

Reply:
Change `Stream`  to `Duplex` not able to find any document for same.